packages/react-form-renderer/src/constants/index.js

```diff
@@ -57,6 +57,7 @@ export const validators = {
 
 export const layoutComponents = {
   FORM_WRAPPER: 'FormWrapper',
+  PLAIN_TEXT: 'Text',
```

Comment:
There is already a plain text component in component mapper

Just FYI we plan to merge layout and formFields mappers into one in v2 which will come with release of react v17

Reply:
Should I do something with it till then?

Reply:
use the current plain text component please

Reply:
I needed a different type than text (because this has to be wrapped in input addon element) so I renamed it to reflect the function of it to INPUT_ADDON_TEXT.

Reply:
> because this has to be wrapped in input addon element

Can it be done automatically?

Reply:
I am afraid that we will end up with special copies of some elements like ICON, BUTTON etc, just to put them inside input addon.

Reply:
So you suggest that we create an INPUT_ADDON and INPUT_ADDON_BUTTON and leave the others? (There is also a problem with the BUTTON since it has an inline styling margin-left 3px which breaks the input addon style, this is why I used another one.)

Reply:
> So you suggest that we create an INPUT_ADDON and INPUT_ADDON_BUTTON and leave the others? 

yes

> (There is also a problem with the BUTTON since it has an inline styling

Just pass the className (pf has classname for inline button) as a prop to the button. It does accept outside props.

Reply:
The class name won't work. This is what is rendered under BUTTON:
`const ButtonOverride = (props) => <Button style={{ marginLeft: 3 }} { ...props } />;
` which is nearly inpossible to overwrite, and for sure a bad pattern.